Modules/Core/Common/include/itkMersenneTwisterRandomVariateGenerator.h

```diff
@@ -204,7 +204,7 @@ class ITKCommon_EXPORT MersenneTwisterRandomVariateGenerator : public RandomVari
 
   /** Get an integer variate in [0, n] for n < 2^32 */
   IntegerType
-  GetIntegerVariate(const IntegerType & n);
+  GetIntegerVariate(const IntegerType n);
```

Comment:
I'm still thinking, shouldn't we explicitly use `uint32_t`, instead of "IntegerType". It appears essential to MersenneTwisterRandomVariateGenerator that the integer type is a 32-bit unsigned integer. 🤷

Reply:
While we might, it is a potentially breaking interface change, whereas the current proposal is not.

Reply:
OK, thanks!

In the future, I hope we will use [std::mersenne_twister_engine](https://en.cppreference.com/w/cpp/numeric/random/mersenne_twister_engine) anyway  😃